migrations/sql/V2019.02.11.10.23__permit_amendments.sql

```diff
@@ -0,0 +1,161 @@
+alter table permit drop constraint permit_pkey;
+ALTER TABLE permit ADD COLUMN permit_id SERIAL PRIMARY KEY;
+alter table mine_party_appt drop constraint mine_party_appt_permit_party_fk;
+
+
+CREATE TABLE permit_amendment_type_code
+(
+    permit_amendment_type_code character varying(2) NOT NULL,
+    description character varying(100) NOT NULL,
+    display_order smallint,
+    active_ind boolean DEFAULT true NOT null,
+    create_user character varying(60) NOT NULL,
+    create_timestamp timestamp
+    with time zone DEFAULT now
+    () NOT NULL,
+    update_user character varying
+    (60) NOT NULL,
+    update_timestamp timestamp
+    with time zone DEFAULT now
+    () NOT NULL
+);
+
+ALTER TABLE permit_amendment_type_code OWNER TO mds;
+ALTER TABLE ONLY permit_amendment_type_code
+
+ADD CONSTRAINT permit_amendment_type_code_pkey PRIMARY KEY
+(permit_amendment_type_code);
+
+INSERT INTO permit_amendment_type_code
+	(
+	permit_amendment_type_code,
+	description,
+	display_order,
+	create_user,
+	update_user
+	)
+VALUES
+	('AM', 'Permit Amendment', 10, 'system-mds', 'system-mds'),
+	('ML', 'Amalgamated Permit', 20, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+CREATE TABLE permit_amendment_status_code
+(
+	permit_amendment_status_code character varying(2) NOT NULL,
+	description character varying(100) NOT NULL,
+	display_order smallint,
+	active_ind boolean DEFAULT true NOT null,
+	create_user character varying(60) NOT NULL,
+	create_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+	update_user character varying(60) NOT NULL,
+	update_timestamp timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE permit_amendment_status_code OWNER TO mds;
+ALTER TABLE ONLY permit_amendment_status_code
+ADD CONSTRAINT permit_amendment_status_code_pkey PRIMARY KEY
+(permit_amendment_status_code);
+
+INSERT INTO permit_amendment_status_code
+	(
+	permit_amendment_status_code,
+	description,
+	display_order,
+	create_user,
+	update_user
+	)
+VALUES
+	('A', 'Active', 10, 'system-mds', 'system-mds'),
+	('R', 'Remitted', 20, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+CREATE TABLE permit_amendment
+(
+	permit_amendment_id serial PRIMARY KEY,
+	permit_amendment_guid uuid DEFAULT gen_random_uuid() NOT NULL,
+	permit_id integer NOT NULL,
+	received_date date,
+	issue_date date,
+	authorization_end_date date,
+	permit_amendment_type_code character varying(2) DEFAULT 'W' not null ,
```

Comment:
What code value is "W"?

Reply:
W was originally a Work amendment, and there were Work and Non-Work types. Missed changing the default when that was identified as not required. Good catch, thanks.

Reply:
Updated to default to AMD